lib/msal-native-auth/src/auth_flow/result/GetAccessTokenResult.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { AuthenticationResult } from "@azure/msal-browser";
+import { ResultBase } from "./ResultBase.js";
+import { GetAccessTokenState } from "./AuthFlowState.js";
+
+/*
+ * Result of getting an access token.
+ */
+export class GetAccessTokenResult extends ResultBase<
```

Comment:
I would suggest to remove the `Get` this is a class definition.

Reply:
Please check my previous comment. `GetAccessToken` is a action. This class is for the result of that action.